tools/clang/unittests/HLSL/VerifierTest.cpp

```diff
@@ -193,6 +194,21 @@ TEST_F(VerifierTest, RunArrayIndexOutOfBounds) {
   CheckVerifiesHLSL(L"array-index-out-of-bounds-HV-2016.hlsl");
 }
 
+TEST_F(VerifierTest, RunShaderMismatch) {
```

Comment:
If we can group the shader attribute tests into one test called `shader_attribute.hlsl`, then we can call the group a matching name `RunShaderAttribute`, which fits the pattern for the rest of the verifier tests.  Plus, one test group for one file keeps with the existing pattern, and won't break VerifierHelper.py (even though some don't want to keep that alive anyway).